packages/runtime-core/src/components/Teleport.ts

```diff
@@ -80,6 +80,13 @@ export const TeleportImpl = {
 
     const disabled = isTeleportDisabled(n2.props)
     const { shapeFlag, children } = n2
+    // Verify whether the target is an SVGElement or not.
+    const targetIsSVG = (target: RendererElement) => {
```

Comment:
Maybe you can use `isSVGTag(target.tag)` instead this function.

Reply:
👍